src/picotorrent/torrentdetails/torrenttrackerswidget.cpp

```diff
@@ -67,6 +67,9 @@ void TorrentTrackersWidget::onAddTracker()
     dlg->setWindowTitle(i18n("add_tracker"));
     dlg->open();
 
+    QClipboard* clipboard = QGuiApplication::clipboard();
+    dlg->setText(clipboard->text());
```

Comment:
Perhaps we should be a bit restrictive here? Perhaps check if the string looks like a URL - ie. starts with http(s) or udp?